scanpy/preprocessing/recipes.py

```diff
@@ -2,6 +2,7 @@
 """
 
 from . import simple as pp
+from ._deprecated.highly_variable_genes import filter_genes_dispersion
```

Comment:
I guess the `recipe_weinreb17` was relying on `filter_genes_cv_deprecated`, it's unfortunately not tested, but can you check and maybe add a small test? I fear this would break a lot of people's code that used the weinreb recipe.

Reply:
Oh, actually about this recipy, i forgot to import filter_genes_cv_deprecated from ._deprecated.highly_variable_genes. Fixed it.

For the recipies i just changed the point of import, no other things were changes.

Reply:
https://github.com/theislab/scanpy/pull/330/commits/58323228ca2c040723980d5c508e9a4a21e45dbc

Reply:
yes, I was referring to that omission :)